### PR TITLE
Update Territory & Customer Group across all transaction

### PR DESCRIPTION
**Pull request by Zlash65**
update-transaction

Issue - https://github.com/frappe/erpnext/issues/10542

### CODE REVIEW

File: erpnext/patches/v10_0/update_territory_and_customer_group.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import frappe
+from erpnext.selling.doctype.customer.customer import update_linked_doctypes
+
+def execute():
+    customers = frappe.get_all('Customer')
```
Comment (rmehta) [inline erpnext/patches/v10_0/update_territory_and_customer_group.py]: tabs!

File: erpnext/selling/doctype/customer/customer.py [hunk 2]
```diff
@@ -53,6 +54,16 @@ def validate(self):
 		self.flags.old_lead = self.lead_name
 		validate_party_accounts(self)
 		self.validate_credit_limit_on_change()
+		self.check_cgroup_or_terr_change()
+
+	def check_cgroup_or_terr_change(self):
```
Comment (rmehta) [inline erpnext/selling/doctype/customer/customer.py]: never use abbreviations like this!!

https://github.com/frappe/erpnext/wiki/Naming-Guidelines

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/selling/doctype/customer/customer.py [hunk 3]
```diff
@@ -320,4 +336,49 @@ def get_customer_primary_address(doctype, txt, searchfield, start, page_len, fil
 		""", {
 			'customer': customer,
 			'txt': '%%%s%%' % txt
-		})
\ No newline at end of file
+		})
+
+def update_linked_doctypes(parent, child, name, value):
```
Comment (rmehta) [inline erpnext/selling/doctype/customer/customer.py]: move this util function to frappe

**Comment by Zlash65** [comment]
Travis failing because of the functions used that aren't yet merged in the Frappe.
https://github.com/frappe/frappe/pull/5048

**Pull request merged**